### DISCUSSION TITLE
悬浮候选框内，候选项的对齐貌似有点问题。

### DISCUSSION THREAD

**Discussion by Hhsy083212** [fcitx5-android/fcitx5-android#616] [General | 讨论]
目前的功能已经很完美了，知道开发者大佬们都很忙，本来不太想打扰大家，可是我有一丢丢强迫症，也希望F5A越来越好，所以还是忍不住提了出来。😂😂
![Screenshot_2024-11-11-01-25-34-641_com.miui.notes.jpg](https://github.com/user-attachments/assets/1d802a94-d3f2-4bec-9272-71c452da4c52)

![Screenshot_2024-11-11-01-33-42-662_com.miui.notes.jpg](https://github.com/user-attachments/assets/9aa91aeb-f923-43d1-a639-52a20415f1ac)

如图所示，悬浮候选框内，第一个候选项和底下的几个，貌似不对齐，稍微往前了一丢丢，这个也没有任何选项可以调整，应该是BUG吧？🤔
或者我猜测，第一个候选项有高亮背景（虽然这个背景完全无法移动，也貌似没啥意义），是不是第一个候选项的对齐参照的是那个高亮背景呢？
既然高亮背景无法移动，我觉得干脆不要背景了，反正用数字键选词，高不高亮无所谓啦。
呃，大概就是这样。😂😂

**Comment by rocka** [comment]
“高亮背景”可以按 <kbd>Tab</kbd> 键向后移动， <kbd>Shift</kbd>+<kbd>Tab</kbd> 向前移动，按空格上屏当前选中的词。

文字看上去没对齐的原因大概是你用的字体中数字 1 和其他数字的宽度不一致。你可以试一下在输入框里打两行字

```
1. 第一行
2. 第二行
```

看看实际的显示效果能否对齐。

**Reply by Hhsy083212** [reply]
天呐，我试了一下，还真是这样啊，不论是浏览器，还是备忘录，再或者文稿软件，只要打出数字列表，还真是对不齐，小米默认的字体，居然还能出现这种BUG，难怪被称为BUGUI🤣🤣，骂得不亏，不过还好，这个无关紧要，谢谢大佬回复。🙏🙏

目前设置页面里，悬浮窗相关还是英文的，等到正式1.0版本发版时，会有翻译吗？

**Reply by rocka** [reply]
会有的
